### DISCUSSION TITLE
Invalid businessId (validation-report endpoint)

### DISCUSSION THREAD

**Discussion by JasperBeaurain** [OP-TED/eForms-SDK#240] [Q&A] [answered]
Hi,

When I submit a notice on the preview environment, I receive a success message and a link to get the validation report,, e.g.: https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/65667997-031a-4198-bd25-7225449ef414/validation-report

The service always answers with Invalid business id:
400 Status code
```json
{
  "code": "notice.INVALID_BUSINESS_ID",
  "message": "Invalid business id=65667997-031a-4198-bd25-7225449ef414",
  "showDetails": false
}
```

Is this a problem on your side?

Kind regards,
Jasper

**Comment by pcolast** [comment, marked as answer]
Hello Jasper,

We confirm the bug. Currently, the cvs link in the answer is not correct, there is only noticeId (without notice version Id).
To get  the validation report you need to use  65667997-031a-4198-bd25-7225449ef414-01

Regards,
Stefan
